examples/platform/linux/AppMain.cpp

```diff
@@ -165,6 +181,11 @@ void SetupSignalHandlers()
     signal(SIGTERM, OnSignalHandler);
     signal(SIGIO, OnSignalHandler);
     signal(SIGINT, OnSignalHandler);
+
+    signal(SIGUSR1, OnSignalHandler);
+    signal(SIGUSR2, OnSignalHandler);
+    signal(SIGHUP, OnSignalHandler);
+    signal(SIGTTIN, OnSignalHandler);
```

Comment:
Thanks for moving this out of the platform layer and into the app.

It was very unkind for the sdk to be using signals in the platform layer, as that's not really an acceptable thing for the sdk to do when it is being linked to an existing application as a library.